src/platform/plugins/shared/discover/public/ebt_manager/scoped_discover_ebt_manager.ts

```diff
@@ -333,4 +355,22 @@ export class ScopedDiscoverEBTManager {
       },
     };
   }
+
+  public trackTabs({
```

Comment:
Tiny nit for consistency:
```suggestion
  public trackTabsEvent({
```

Reply:
addressed in [77f013e](https://github.com/elastic/kibana/pull/234344/commits/77f013ec48732562975b631d1466a9f3ac6b0348)